kaedama.py

```diff
@@ -54,6 +54,15 @@
 
 arg_parser.add_argument( '--print-query',dest='printquery',help="Print the query after parameter substitution and exit", action="store_true", default=False )
 
+
```

Comment:
Provides a default dictionary which will define the directory layout for the output, log, condor and histogram files.